tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -280,14 +280,14 @@ def submit_command(
 
         return self._submit_batch_job(
             job_submit_command,
-            nodes,
-            slots,
-            host,
-            after_ok,
-            partition,
-            constraint,
-            prefer,
-            other_options,
+            nodes=nodes,
```

Comment:
let's use the same approach in the line 317 too

Reply:
Done